### PR TITLE
refactor(docs): updating button pointerover lightweight styling pic a…

### PR DESCRIPTION
**Pull request by Arieldelossantos**
closes https://github.com/unoplatform/Uno.Themes/issues/1181

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

Updating Docs

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested the changes where applicable:
	- [ ] UWP
	- [ ] iOS
	- [ ] Android
	- [ ] WASM
	- [ ] MacOS
- [ ] Updated the documentation as needed:
	- [ ] [General Doc Update](https://github.com/unoplatform/Uno.Themes/tree/master/doc)
	- [ ] [material-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-controls-styles.md)
	- [ ] [cupertino-controls-styles.md](https://github.com/unoplatform/Uno.Themes/blob/master/doc/cupertino-controls-styles.md)
	- [ ] [lightweight-styling.md (LightWeight Styling Resource Keys)](https://github.com/unoplatform/Uno.Themes/blob/master/doc/lightweight-styling.md)
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: doc/lightweight-styling.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ uid: uno.themes.lightweightstyling
...
+<Button Content="Default Button Style" Style="{StaticResource MaterialFilledButtonStyle}" />
+
+<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+	<Button.Resources>
+		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
+	</Button.Resources>
+</Button>
+
```
Comment (kazo0) [inline doc/lightweight-styling.md:34]: Use this XAML and retake the screenshot:

```xml
<Button Content="Default Button Style"
		Style="{StaticResource FilledButtonStyle}" />

<Button Content="Overridden Button Style"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
	</Button.Resources>
</Button>

<Button Content="Overridden Button Style (PointerOver)"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver" Color="DarkRed" />
		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver" Color="LightPink" />
		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver" Color="DarkRed" />
	</Button.Resources>
</Button>
```

File: doc/assets/material-button-pointerover-lightweight-styling.png
Comment (kazo0) [inline doc/assets/material-button-pointerover-lightweight-styling.png:1]: Code looks good but let's keep the screenshot in Light mode

Comment (Arieldelossantos) [reply doc/assets/material-button-pointerover-lightweight-styling.png:1]: done

File: doc/lightweight-styling.md [hunk 2]
```diff
@@ -22,16 +22,31 @@ uid: uno.themes.lightweightstyling
...
+		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
+		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver"
+							Color="DarkRed" />
+		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver"
+							Color="LightPink" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver"
+							Color="DarkRed" />
 	</Button.Resources>
 </Button>
 ```
```
Comment (kazo0) [inline doc/lightweight-styling.md:52]: Sorry I'm annoying :) but I formatted the XAML a bit better here so can you use this one:

```xml
<Button Content="Default Button Style"
		Style="{StaticResource FilledButtonStyle}" />

<Button Content="Overridden Button Style"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
	</Button.Resources>
</Button>

<Button Content="Overridden Button Style (PointerOver)"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver" Color="DarkRed" />
		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver" Color="LightPink" />
		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver" Color="DarkRed" />
	</Button.Resources>
</Button>
```

No need to change the screenshot, it looks good as is

Comment (Arieldelossantos) [reply doc/lightweight-styling.md]: done

**Review by kazo0 — APPROVED** [review]

**Comment by agneszitte** [comment]
@Mergifyio backport legacy/2x release/stable/3.0

**Pull request merged**